experimental/juvix/contract.jvx

```diff
@@ -0,0 +1,198 @@
+let Map = Data.Map
+
+-- common module interfaces for data structures in Common?
+sig MonoidMeasure : Common.Measure
```

Comment:
What is this declaring, exactly? A type alias?

Reply:
the Map is a module.

MonoidMeasure, is a module, so I'm stating the module type.

Reply:
Gotcha, makes sense.